src/components/form-element/index.js

```diff
@@ -10,7 +10,7 @@ const InputForm = ({ value, readOnly, onChange, ...otherProps }) => {
   return (
     <>
       {
-        readOnly ? <span>{ value }</span> : (
+        readOnly ? <span>{ value?value:'—' }</span> : (
```

Comment:
Instead of ternary operator, you can make use of OR operator.

so instead of (value ? value : "--") => make it (value || "--")